docs/content/tutorials/helm/index.md

```diff
@@ -0,0 +1,652 @@
+secret/redis-secret created
+
+> helm upgrade demo ./Chart -n demo --install
+Release "demo" does not exist. Installing it now.
+NAME: demo
+LAST DEPLOYED: Wed Sep 13 01:05:19 2023
+NAMESPACE: demo
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+```
+
+{{< alert title="⚠️ Chart Directory" color="warning" >}}
+If you see an error message like **Error: path "./Chart" not found** then you are in the wrong directory. Make sure your terminal is in the `./demo` directory of the `samples` repository.
+{{< /alert >}}
+
+Run the following command to check if everything is running:
+
+```bash
+kubectl get all -n demo
+```
+
+The output should look similar to the following:
+
+```txt
+> kubectl get all -n demo
+NAME                       READY   STATUS    RESTARTS   AGE
+pod/webapp-79d5dfb99-vhj9g   1/1     Running   0          2m48s
+
+NAME                   READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/webapp   1/1     1            1           2m49s
+
+NAME                             DESIRED   CURRENT   READY   AGE
+replicaset.apps/webapp-79d5dfb99   1         1         1       2m49s
+```
+
+> The generated names and ages of the objects will be different in your output. Make sure you see the status of `Running` for the `pod/webapp-...` entry. If the status is not `Running`, try repeating the `kubectl get all -n demo` after waiting.
+
+At this point you've deployed the application but you have not actually used Radius yet. You will start doing that in the next step, as well as set up and use Redis.
+
+The steps so far are similar to how many applications are managed today:
+
+- ⬜️ Dependencies like Redis are provisioned manually and separately from application deployment.
+- ⬜️ Connection information like passwords and addresses is manually stored in secret stores.
+- ⬜️ Applications access the connection information from those secret stores when they are deployed.
+
+Over the next few steps you will update this application to use Radius so that:
+
+- ✅ Dependencies like Redis are provisioned on-demand when they are needed.
+- ✅ Connection information is managed automatically, secret stores are an implementation detail.
+- ✅ Applications have a documented relationship with the dependencies they connect to.
+
+From here you will go through a series of steps to incrementally add more Radius features to the application.
+
+## Step 4. Add Radius
+
+Make sure the `app.yaml` file from `./demo/Chart/templates/app.yaml` is open in your editor. You will make some edits to this file to enable Radius.
+
+Add the `annotations` property to `metadata`, and then add the `radapp.io/enabled: 'true'` annotation. The `'true'` must be surrounded in quotes.
+
+The example below shows the updated `metadata` section after making the changes.
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: webapp
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    radapp.io/enabled: 'true'
```

Comment:
Can we find a way to highlight this similar to how we highlighted code in the getting started guide?

Reply:
trying this to see if it'll work - might be tricky to format within a code block

Reply:
looks like it's a tradeoff - can add highlighting on top of language based syntax highlighting...so I'll opt for an inline comment to make this clear.